lms/djangoapps/instructor_task/__init__.py

```diff
@@ -1,2 +1 @@
 # lint-amnesty, pylint: disable=missing-module-docstring
-default_app_config = 'lms.djangoapps.instructor_task.apps.InstructorTaskConfig'
```

Comment:
Explicit declaration of default_app_config was deprecated in `Django 3.2` [See [release notes](https://docs.djangoproject.com/en/4.2/releases/3.2/#id3)]